src/_gettsim/transfers/kindergeld.py

```diff
@@ -1,11 +1,7 @@
 from _gettsim.shared import dates_active
 
 aggregate_by_group_kindergeld = {
-    "kumulativer_kindergeld_anspruch_tu": {
-        "source_col": "kindergeld_anspruch",
-        "aggr": "cumsum",
```

Comment:
Please get rid of the cumsum aggregator then! (S+R for cumsum in entire project, remove all related code)